### ISSUE TITLE
element-wise add in dnn module

### ISSUE THREAD

**Issue by TouqeerAhmad** [opencv/opencv#14001]
##### System information (version)

- OpenCV => :grey_question:
- Operating System / Platform => :grey_question:
- Compiler => :grey_question:

**Comment by TouqeerAhmad** [comment]
Hi, I wanted to verify if dnn module supports element-wise addition e.g. in tensor flow I can do tf.math.add(in1,in2) where in1 and in2 are two tensors of equal dimensions in NCHW order. I am trying to load my tensor flow model in opencv and as soon as I put the add in my Tensorflow model, opencv/dnn module generate error whereas tf.nn.bias_add works fine on opencv end.
Hope to get an answer on this!
Thanks!

**Comment by dkurt** [comment]
@TouqeerAhmad, please provide a miminal working reproducer. Add TensorFlow code snippet.

**Comment by TouqeerAhmad** [comment]
I was able to work around it. What I was trying to do is to add the output of two conv2d layers (in1,in2) each of dimensions 10x30x30x16 

out = tf.math.add(in1,in2)

Though there is no problem doing addition on the output of conv2d layers in tensor flow; opencv did not like it. I ended up saving the in1 and in2 into two placeholders in tensor flow and then did the tf.math.add on the saved placeholders -- OpenCV accepted that without any issue. 

Thank you, closing the the issue.
